src/main/java/com/eva/model/person/applicant/ApplicationStatus.java

```diff
@@ -13,7 +13,7 @@
     public static final String MESSAGE_CONSTRAINTS =
             "Application Status should only contain the words: processing, accepted, rejected. It should not be blank";
     public static final String[] POSSIBLE_STATUSES = {"received", "processing", "accepted", "rejected"};
```

Comment:
maybe we can use an enum for this

Reply:
Hmm yea, noted about this from previous PR, will try to figure out some time to do this, thanks for reminding!